docs/pages/components/icon/Icon.vue

```diff
@@ -77,7 +89,38 @@
                 Vue.use(Buefy, {
                   defaultIconComponent: 'vue-fontawesome',
                   defaultIconPack: 'fas',
-                });`
+                });`,
+                ExCustomCode: `
```

Comment:
You can remove it and show the example code adding `import ExCustomCode from '!!raw-loader!./examples/ExCustom'` (without | pre)